src/components/Dashboard/Carousel/PreviewPost/PreviewPost.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const PreviewPost = ({ post }) => {
```

Comment:
```suggestion
export const PreviewPost = ({ post }) => {
```

Reply:
I don't know, `TextPost` sounds like the content of the post and each slide contains only a preview of the post with a link to the full Post.

Reply:
in any case, the component should be not in `src/components/Dashboard/Carousel/PreviewPost/PreviewPost.js`,
instead should be in `src/components/Dashboard/PreviewPost/PreviewPost.js`

Reply:
I was thinking in having different kinds of posts like text posts, image posts. But yes they are all like a preview, or small summary of a bigger post in a blog.

Reply:
Then I will rename it to `TextPreviewPost`, because this is kind of like a preview but with no images, or media.

Reply:
ok, I change location and file name